src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -19,6 +19,66 @@ Go here to get or create your [New Relic user key](https://one.newrelic.com/laun
 
 Once you have your user key, in **Observability** click **Connect to New Relic One**, then paste your user key and click **Connect**.
 
+## Monitor code-level metrics [#code-level]
+
+CodeStream's code-level metrics give you detailed insight into how your code is performing.
+
+![A screenshot of a Python method with average duration, throughput, and error rate telemetry.](./images/code-level-metrics.png "A Python method with average duration, throughput, and error rate metrics.")
+
+<figcaption>When relevant, CodeStream's code-level metrics are shown in a CodeLens above your Python methods.</figcaption>
+
+<Callout variant="caution">
```

Comment:
This is fine, but I wonder if we can avoid using a callout here by changeing the title of the section to something like `Montitor code-level metrics (Python only)`

Your call on this one!

Reply:
This is a good suggestion and I wish I didn't feel the need to leave this caution here. Other languages will be added eventually.